DMTN-107.tex

```diff
@@ -417,20 +428,20 @@ \subsection{Catalog Differencing}\label{ssec:potsol_catdiff}
 
 % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % %
 \clearpage
-\section{Summary and Recommendations}\label{sec:rec}
+\section{Summary and Conclusions}\label{sec:rec}
 
-The option for alert production in LOY1 which would provide the largest potential science impact with {\em no} expansion in DM scope is to only use the template images that will have already been generated for the commissioning Mini-Surveys 1 and 2 ($\sim$1600 $\rm deg^2$).
+The option for alert production in LOY1 that would provide the largest potential science impact with {\em no} expansion in DM scope is to only use the template images that will have already been generated for the commissioning Mini-Surveys 1 and 2 ($\sim$1600 $\rm deg^2$).
 A very small up-scope option is to re-generate template images from {\em all} of the commissioning data, which could add at least another $\sim$400 $\rm deg^2$ from the planed surveys described in \citeds{LSE-79}, and potentially 10,000 $\rm deg^2$ if template-building is prioritized during commissioning as proposed in \S~\ref{sssec:potsol_comm_r}.
-A further moderate up-scope option is to continue to generate template images throughout LOY1, as new data is obtained, as this could add $\sim$1000-2000 $\rm deg^2$ per month.
+A further moderate up-scope option is to continue to generate rolling template images throughout LOY1, as new data is obtained, as this could add $\sim$1000-2000 $\rm deg^2$ per month.
```

Comment:
I still don't like "rolling"

Reply:
what about 
- A further moderate up-scope option is to continue to generate prefatory template images

Reply:
resolved by using "prefatory"